io.github.ychclone.blink.yml

```diff
@@ -0,0 +1,29 @@
+id: io.github.ychclone.blink
+runtime: org.kde.Platform
+runtime-version: '6.7'
+sdk: org.kde.Sdk
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+command: blink
+modules:
+  - name: qscintilla
+    buildsystem: qmake
+    subdir: src
+    sources:
+      - type: archive
+        url: https://www.riverbankcomputing.com/static/Downloads/QScintilla/2.13.4/QScintilla_src-2.13.4.tar.gz
+        sha256: 890c261f31e116f426b0ea03a136d44fc89551ebfd126d7b0bdf8a7197879986
+      - type: patch
+        path: qscintilla-lib-paths.patch
+  - name: blink
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+    sources:
+      - type: archive
+        url: https://github.com/ychclone/blink/archive/refs/tags/1.9.1.tar.gz
```

Comment:
I don't see a metainfo file in the archive https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#path-and-filename

The page above has everything you need to create one.

Reply:
Hi, I've added the metainfo in https://github.com/ychclone/blink/archive/refs/tags/1.9.1.tar.gz,
folder /app/share/metainfo

Reply:
Hi, I've updated the sha256